### PR TITLE
Update docs for Java 11

### PR DESCRIPTION
**Pull request by Manno15**
While setting up a new laptop, I noticed the requirements docs were still on Java 8. So just a simple change to java 11.

### CODE REVIEW

File: docs/ubuntu1604.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Ubuntu 16.04 for Uno.
 
 ```bash
 #install stuff
-sudo apt install openjdk-8-jdk maven git openssh-server wget libxml2-utils make g++
+sudo apt install openjdk-11-jdk maven git openssh-server wget libxml2-utils make g++
```
Comment (keith-turner) [inline docs/ubuntu1604.md]: I don't think java 11 is available in the default Ubuntu 16.04 repos.  It is for 18.04.

**Comment by kpm1985** [comment]
Requires a PPA to do 11 afaik all I see for Xenial is OpenJdk9


https://packages.ubuntu.com/hu/xenial/java/

File: docs/ubuntu1604.md (hunk 1 above)
Comment (Manno15) [reply docs/ubuntu1604.md]: Yeah, after some google checks, it appears you are correct. My mistake. Do you think it is best if I create a separate doc for 18.04 or newer and keep this one as it was?

Comment (keith-turner) [reply docs/ubuntu1604.md]: > Do you think it is best if I create a separate doc for 18.04 or newer and keep this one as it was?

I am in favor of that.

**Comment by Manno15** [comment]
Added a quick doc for 18.04. Let me know if I need to make any further changes to it.

**Comment by keith-turner** [comment]
> Added a quick doc for 18.04. Let me know if I need to make any further changes to it.

Did you try running through this on an 18.04 system?

**Comment by Manno15** [comment]
I do not have a machine that specifically has 18.04 on it but I confirmed online that the command is correct and the rest is still working on Pop OS which is based on 19.04. I'

**Comment by cjmctague** [comment]
I will spin up a VM in my lab and test this

**Comment by keith-turner** [comment]
> I will spin up a VM in my lab and test this

@cjmctague did you ever get a chance to try that

**Comment by cjmctague** [comment]
Was able to setup and run Uno on 18.04. LGTM

**Pull request merged**